### ISSUE TITLE
🪲 Bug Report - "Show/Hide Query/Script" code may show/hide other section's code

### ISSUE THREAD

**Issue by ehaslett** [Azure/Azure-Proactive-Resiliency-Library#134]
## Describe the bug

Using hugoIf multiple KQL queries contain exactly the same code, the code snippet in _index.md will show/hide _all_ of the KQL code sections when any of the corresponding Show/Hide Query/Script buttons are used.

For example, when WAF-1, WAF-2, WAF-4 and WAF-5  all have the same code (in separate files) of "// cannot-be-validated-with-arg", then any of the Show/Hide Query/Script buttons will toggle showing/hiding all of the code sections for WAF-1, WAF-2, WAF-4, WAF-5.

Live example:
https://azure.github.io/Azure-Proactive-Resiliency-Library/services/networking/firewall/

Toggle any of AFW-2, AFW-4 or AFW-7 to show/hide all 3 at the same time.

## To Reproduce

Steps to reproduce the behaviour:

1. Create KQL code files with the same content
2. Using either hugo or the live site with the updated code, toggle the corresponding sections
3. Note if all affected sections can toggle the state of all other affected sections.

## Expected behaviour

A section's Show/Hide Query/Script toggle button should only show/hide that particular section's code.

## Screenshots 📷

![Screenshot 2023-10-30 171817](https://github.com/Azure/Azure-Proactive-Resiliency-Library/assets/60371213/1b9e0e38-cfa7-44d6-a7a9-dff25337b51b)

## Additional context

Anything else we should know to help us troubleshoot this bug?

**Comment by oZakari** [comment]
Thanks for calling this out @ehaslett, I work on getting this fixed in the next sprint.

**Comment by oZakari** [comment]
@ehaslett After investigating this further, we determined that there are some limitations with how APRL was originally setup within Hugo. Also, there are nested shortcodes being used which limit our ability to establish a unique identifier without manually adding a unique ID for each query. As such, we decided that the time required to resolve this was not worth it due to APRL eventually going to be deprecated.
